snap/gui/oudedetai.logos.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Logos Bible
+GenericName=FaithLife Wine App Installer
+Comment=Manages FaithLife Bible Software via Wine (snap)
+Exec=oudedetai.logos
```

Comment:
Where is this stored?

Reply:
same reply as for `oudedetai.verbum.desktop` above.